cli/internal/cache/cache.go

```diff
@@ -271,6 +277,24 @@ func (mplex *cacheMultiplexer) Fetch(target string, key string, files []string)
 	return false, files, 0, nil
 }
 
+func (mplex *cacheMultiplexer) Exists(target string) (CacheState, error) {
+	syncCacheState := CacheState{}
+	for _, cache := range mplex.caches {
+		cacheState, err := cache.Exists(target)
+		if err != nil {
+			return syncCacheState, err
+		}
+		if cacheState.Local == true {
```

Comment:
I should've been more explicit here: you don't need the `if` statements anymore. If the relevant part of `cacheState` is `true`, it'll flip the `syncCacheState` boolean to `true`, and if it's `false`, it won't do anything. So no need to check beforehand

Reply:
doh - sorry.. totally didn't catch that.

Reply:
fixed